model/src/test/java/uk/gov/pay/commons/api/json/PayZonedDateTimeSerializerTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.commons.api.json;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.time.ZonedDateTime;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class PayZonedDateTimeSerializerTest {
+    private PayZonedDateTimeSerializer serializer;
+
+    @Before
+    public void before() {
+        serializer = new PayZonedDateTimeSerializer();
```

Comment:
This can be initialised in the field declaration so we don’t need the `@Before` method at all:

```java
private final PayZonedDateTimeSerializer serializer = new PayZonedDateTimeSerializer();
```